routes/cloudinary-routes.js

```diff
@@ -0,0 +1,19 @@
+import cloudinary from '../services/cloudinary';
+
+function uploadAsset(req, res) {
+    const imageUrl = "https://upload.wikimedia.org/wikipedia/commons/a/ae/Olympic_flag.jpg";
```

Comment:
What does this link do?
Did you test it from the frontend?

Reply:
it is for uploading asset, i have not tested it

Reply:
<img width="764" alt="image" src="https://github.com/WeFriiends/WeFriiendsProfile/assets/107632181/1d58cbb5-40b1-49a0-ad93-8bba2c04d784">